src/index.ts

```diff
@@ -0,0 +1,51 @@
+// =======================
+// Domain: Event publish/subscribe
+// =======================
+
+import sbp from '@sbp/sbp'
+import '@sbp/okturtles.data'
+
+const listenKey = (evt: string) => `events/${evt}/listeners`
+
+type ThisType = {
+  errorHandler?: (event: string, e: unknown) => void
+}
+
+export default (sbp('sbp/selectors/register', {
+  'okTurtles.events/_init': function (this: ThisType) {
```

Comment:
SBP automatically assigns `this`, it shouldn't be passed in here like this.

Reply:
This doesn't pass or define `this`. It defines the type for `this`.

Reply:
Can you elaborate please? Is this some TypeScript weirdness?

In regular JS this would be be a function parameter called `this`....

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#specifying-the-type-of-this-for-functions